src/main/java/com/google/devtools/build/lib/BUILD

```diff
@@ -100,7 +100,7 @@ java_library(
         "//conditions:default": ["//src/main/native:libunix.so"],
     }),
     deps = [
-        "//src/main/java/com/google/devtools/build/lib:os_util",
+        ":os_util",
```

Comment:
Why all these no-op changes here? I mean, I guess they're fine, but they make it harder to see what actually changed :)

Reply:
Sorry, I just wanted to shorten my new target, but my regex was a little greedy and fixed all of the local targets. I can revert everything except command-utils if you'd like.